infrahub-sync/infrahub_sync/cli.py

```diff
@@ -136,7 +137,7 @@ def generate(
     client = InfrahubClientSync(address=infrahub_address)
 
     try:
-        schema = client.schema.all()
+        schema = client.schema.all(branch=branch)
```

Comment:
According to the above line you could specify the branch in a file as well?

Reply:
We could load it from the file if we want to allow extra settings in the sync config class. Let’s do it in another feature

Reply:
you are right! I just update the code to allow it